homeassistant/components/recorder/websocket_api.py

```diff
@@ -497,21 +491,30 @@ def ws_info(
     hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict[str, Any]
 ) -> None:
     """Return status of the recorder."""
-    instance = get_instance(hass)
-
-    backlog = instance.backlog if instance else None
-    migration_in_progress = async_migration_in_progress(hass)
-    migration_is_live = async_migration_is_live(hass)
-    recording = instance.recording if instance else False
-    thread_alive = instance.is_alive() if instance else False
+    if instance := get_instance(hass):
```

Comment:
There was no coverage for this because all of the ternary operators made the individual lines covered but no branch coverage.

The case where instance is None did not actually work because `instance.max_backlog` was called on `None`